test/Constraints/noderivative.swift

```diff
@@ -0,0 +1,25 @@
+// RUN: %target-typecheck-verify-swift
+
+import _Differentiation
```

Comment:
Because the test is not in the `test/AutoDiff` folder, it needs `REQUIRES: differentiable_programming` or people that build without that feature will fail this test.

Reply:
@asavonic Please submit a follow-up PR

Reply:
I'm sorry for causing this issue. I've submitted a PR to fix it: https://github.com/apple/swift/pull/67190